docs/detections/rules-ui-create.asciidoc

```diff
@@ -316,6 +316,10 @@ the rule. For example, links to background information.
 .. *False positive examples* (optional): List of common scenarios that may produce
 false-positive alerts.
 .. *MITRE ATT&CK^TM^ threats* (optional): Add relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.
+.. *Custom highlighted fields* (optional): Specify highlighted fields for personalized alert investigation flows. Fields with data are added to the [investigation-section,Highlighted fields] section within the alert details flyout. You can also find custom highlighted fields in the About section of the rule details page.
```

Comment:
Is "Fields with data are added..." meant to imply that fields with no values wouldn't be added to these sections? If so, would it be beneficial to state this explicitly?